src/BrazeKit-dev.js

```diff
@@ -287,6 +354,95 @@ var constructor = function () {
         }
     }
 
+    // mParticle commerce events use different appboy methods depending on if they are
+    // a purchase event or a non-purchase commerce event
+    function logCommerceEvent(event) {
+        var forwardProductsWithEvent =
```

Comment:
Do we need to check this every time someone logs an event? Since it's coming from the settings, I think this variable can be moved up globally to be assigned on init.

Reply:
Yes good point